src/jaxsim/high_level/model.py

```diff
@@ -949,6 +949,41 @@ def com_position(self) -> jtp.Vector:
     # Algorithms
     # ==========
 
+    @functools.partial(oop.jax_tf.method_ro)
+    def coriolis_matrix(self) -> jtp.Matrix:
+        from jaxsim.physics.algos.coriolis import coriolis
+
+        M, M_dot, C = jaxsim.physics.algos.coriolis.coriolis(
+            model=self.physics_model,
+            q=self.data.model_state.joint_positions,
+            qd=self.data.model_state.joint_velocities,
+            xfb=self.data.model_state.xfb(),
+        )
+
+        def body(C):
+            W_H_B = self.base_transform()
+            B_X_W = sixd.se3.SE3.from_matrix(W_H_B).inverse().adjoint()
+            C = B_X_W.T @ C @ B_X_W
```

Comment:
I still haven't run this code, but I was expecting a matrix $C \in \mathbb{R}^{(6+n)\times(6+n)}$. Here it seems be $\mathbb{R}^{6 \times 6}$, am I missing something?

Reply:
To be honest, I do not remember if this transformation ensures that the \dot{M} = C + C^T is preserved, but it should be easy to check (we can do this at the whiteboard next week).